File: Beecrowd/1182 Coluna na Matriz/main.cpp
#include <iostream>

using namespace std;

int main()
{
    const int TAM = 12;
    double M[TAM][TAM], soma = 0, media = 0;
    int i, j, C;
    char T;

    cin >> C;
    do
    {
        cin >> T;
    } while (T != 'S' && T != 'M');

    for (i = 0; i < TAM; i++)
    {
        for (j = 0; j < TAM; j++)
            cin >> M[i][j];
    }

    for (i = 0; i < TAM; i++)
        soma += M[i][C];

    cout << fixed;
    cout.precision(1);
    if (T == 'S')
        cout << soma << endl;
    else
    {
        media = soma / TAM;
        cout << media << endl;
    }
    return 0;
}

File: Beecrowd/1182 Coluna na Matriz/main.py
M = [[0,0,0,0,0,0,0,0,0,0,0,0], [0,0,0,0,0,0,0,0,0,0,0,0], [0,0,0,0,0,0,0,0,0,0,0,0],
     [0,0,0,0,0,0,0,0,0,0,0,0], [0,0,0,0,0,0,0,0,0,0,0,0], [0,0,0,0,0,0,0,0,0,0,0,0],
     [0,0,0,0,0,0,0,0,0,0,0,0], [0,0,0,0,0,0,0,0,0,0,0,0], [0,0,0,0,0,0,0,0,0,0,0,0],
     [0,0,0,0,0,0,0,0,0,0,0,0], [0,0,0,0,0,0,0,0,0,0,0,0], [0,0,0,0,0,0,0,0,0,0,0,0],]
soma = 0
media = 0
TAM = 12
T = 's'

C = int(input())

while T != 'S' and T != 'M':
  T = input()

for i in range(TAM):
  for j in range(TAM):
    M[i][j] = float(input())
  
for i in range(TAM):
  soma += M[i][C]

if T == 'S':
  print("%.1lf" %soma)
else:
  media = soma / TAM
  print("%.1lf" %media) 